portsyncd/linksync.cpp

```diff
@@ -195,6 +196,10 @@ void LinkSync::onMsg(int nlmsg_type, struct nl_object *obj)
         if (!g_init && g_portSet.find(key) != g_portSet.end())
         {
             g_portSet.erase(key);
+            FieldValueTuple tuple("state", "TRUE");
+            vector<FieldValueTuple> vector;
+            vector.push_back(tuple);
+            m_stateTableProducer.set(key, vector);
```

Comment:
The problem I see here is that it will update stateDB port table with ":" separator which is inconsistent with configDB.
It is a general problem, due to table separator is changed to "|" from ":" by sonic-py-swsssdk,  all other C++ swss-common library which has fixed ":" table separator won't be able to work with configDB.

Reply:
we should not use stateTableProducer, because we will not use stateTable Consumer  to consume the data as it only supports one consumer.

Reply:
I'll use Table class for now.

Reply:
Could you update the change with latest Table class and use "|" for port state table in stateDB?